frontend/packages/pipelines-plugin/src/components/catalog/const.ts

```diff
@@ -0,0 +1 @@
+export const TEKTON_HUB_API_ENDPOINT = 'https://api.hub.tekton.dev';
```

Comment:
isn't this endpoint id there within operator, so that we can read and could be dynamic

Reply:
Backend support for reading the endpoints is not there yet, so for now we are going with tektonhub API